ansible/roles/wordpress-instance/tasks/main.yml

```
@@ -108,3 +108,12 @@
   tags:
     - convert-shortcode-to-block
 
+- name: Sync menus
```

Si on considère cette entrée comme un groupement de tâches, peut-on l'appeller 
```suggestion
- name: Set up menus
```